code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,237 @@
+
+// Player Verbs
+
+
+/client/verb/check_exp()
+	set name = "EXP"
+	set desc = "Shows your EXP status"
+	set category = "Special Verbs"
+	//if(!establish_db_connection())
+	//	return 0
+	var/body = "<html><head><title>EXP for [key]</title></head><BR>"
+	body += get_exp_report()
+	body += "</HTML>"
+	src << browse(body, "window=playerexp;size=550x615")
+
+
+// Admin Verbs
+
+/client/proc/cmd_mentor_check_player_exp()	//Allows mentors / admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player XP"
+	if(!check_rights(R_MENTOR|R_MOD|R_ADMIN))
```

Comment:
cmd_mentor_check_new_players is an existing, mentor-accessible command which shows mentors who the new players are on the server, based on # of days since account creation.

cmd_mentor_check_player_exp is the exact same thing, just based on XP/playtime instead of days since registration. I can make it admin-only if that's what people want, but personally, a "show me people new to the server" command seems like exactly the sort of thing that would help Mentors identify and help new players.